shell/client/_topbar.scss

```diff
@@ -459,9 +436,43 @@ body>.topbar {
 
   >.navbar {
     @media #{$desktop} {
-      &.hide-desktop {
+      button.navbar-shrink {
+        @extend %unstyled-button;
+        display: block;
+        width: 100%;
+        height: $navbar-shrink-button-height;
+        border: 0px solid #303030;
+        border-left-width: ($navbar-width-desktop - $navbar-width-desktop-shrunk);
+        background-color: #606060;
+        color: #ddd;
+        text-align: center;
+        line-height: $navbar-shrink-button-height;
+        padding: 0;
+        outline: none;
```

Comment:
Hmm, I forgot I did that. I guess I probably added it because I personally find it really obnoxious when these outlines appear around a button when I click on it. It's not particularly rational but it triggers my OCD senses -- like leaving a room with the light still on. I find I'm compelled to click somewhere neutral to make the outline go away.

I've removed this line for now, but I hope we can find a solution that doesn't involve outlines or box shadows. Or maybe we can find a solution where clicking on a thing doesn't cause its focus indicator to turn on, while keyboarding to it does.